pulsar-client/src/main/java/org/apache/pulsar/client/impl/conf/ClientConfigurationData.java

```diff
@@ -105,6 +109,11 @@
     @JsonIgnore
     private Clock clock = Clock.systemDefaultZone();
 
+    // socks5
+    private InetSocketAddress socks5ProxyAddress;
```

Comment:
Could using the `String` type? We could generate the `InetSocketAddress` in the `PulsarChannelInitializer.java ` when initializing the socks5 handler.

Reply:
I just feel it's not accurate by string. If defined in String, mostly have to parse(like System property).
BTW,  it's the same definition with the Socks5ProxyHandler in Netty.

Reply:
ok